.github/workflows/automatic.yml

```
@@ -62,50 +43,37 @@
           ./scripts/test_docker_image.sh -t "sha-${{ steps.version.outputs.short_sha }}" --basic --discord-token "${{ secrets.FAKE_DISCORD_TOKEN }}"
-
           echo "Docker image test passed successfully!"
 
       - name: Build and Push Docker image with tags
-        uses: docker/build-push-action@v5
+        uses: ./.github/actions/common-docker-build
         with:
-          context: .
-          push: true
+          push: "true"
           tags: ${{ steps.version.outputs.docker_tags }}
```

actionlint (1.7.4)

property "docker_tags" is not defined in object type {}

(expression)